popup.js

```diff
@@ -5,51 +5,112 @@ document.addEventListener('DOMContentLoaded', function () { // this function  st
     var token = localStorage.getItem('trello_token');
     var board_id = localStorage.getItem('board_id');
     var create_board = document.getElementById("create_board");
+    var use_existing_board = document.getElementById("use_existing_board");
+    var board_url = document.getElementById("board_url");
+    var board_url_div = document.getElementById("board_url_div");
+    var set_board = document.getElementById("set_board");
+    var board_url_error = document.getElementById("board_url_error");
     var board_missing_div = document.getElementById('board_missing');
     var oauth_ok_div = document.getElementById('oauth_ok');
     var checkPageButton = document.getElementById('checkPage');
     var today = new Date();
     const monthNames = ["January", "February", "March", "April", "May", "June",
         "July", "August", "September", "October", "November", "December"
     ];
+    const trelloBoardUrlPattern = /https\:\/\/trello\.com\/b\/.*/i;
+
+    create_board.addEventListener('click', function () {
+        board_url_div.style.display = "none";
+        clear_board_url_data();
+    });
+
+    use_existing_board.addEventListener('click', function () {
+        board_url_div.style.display = "block";
+    });
 
     // if token doesn't exist, go to options page and make the user authorize it
     if (!token) {
         chrome.tabs.create({ url: chrome.extension.getURL('settings/index.html') });
         sendResponse();
         return true;
     }
-    // if board does not exist, create one and add it to local storage
+    // if board does not exist, set up one and add it to local storage
     else if (!board_id) {
-        board_create_function();
+        board_set_up_function();
     }
     else {
         working();
     }
 
-    function board_create_function() {
+    function board_set_up_function() {
         board_missing_div.style.display = 'block';
         oauth_ok_div.style.display = 'none';
 
-        create_board.addEventListener('click', function () {
+        set_board.addEventListener('click', function () {
+            document.getElementById("set_board").innerHTML = "preparing your board...";
+
+            if (create_board.checked) {
+                board_create_function();
+            } else {
+                board_use_existing_function();
+            }
+        })
+    }
+
+    function board_create_function() {
+        Trello.post(`/boards?token=${token}&name=Full time Hunt`)
```

Comment:
Looks like this PR got overwritten with your code: https://github.com/HarshdipD/job-tracker/pull/26/files
Could you please fix this?